packages/@aws-cdk-testing/cli-integ/resources/cdk-apps/app/app.js

```diff
@@ -444,6 +444,36 @@ class LambdaHotswapStack extends cdk.Stack {
   }
 }
 
+class DetailedStatusStack extends cdk.Stack{
```

Comment:
none of these resources currently cause the stack itself to emit the `CONFIGURATION_COMPLETE` event, so this test needs to be updated. The only resource I know of that does make it emit this event is ECS Services